[docs] add reference and link to support site (#10825)
yugabyte/yugabyte-db: docs/content/latest/troubleshoot/overview.md

Inspect the YugabyteDB logs for more details on your issue. For more details on where to find and how to understand the YugabyteDB log files, see [Inspect logs](../nodes/check-logs).

> ## 4. Explore Support KB's 
> 
> Additional troubleshooting resources, and information on errata can be found on our [Support page](https://support.yugabyte.com).
> 
> 

Reviewer: ```suggestion
## 4. Explore knowledge base articles

You can find additional troubleshooting resources, and information on errata, on our [Support page](https://support.yugabyte.com).

```
